configs/sandpack-contents/component-theming/slider.js

```diff
@@ -0,0 +1,111 @@
+module.exports = {
+  App: `import {
+        Slider,
+        SliderTrack,
+        SliderFilledTrack,
+        SliderThumb,
+        Text,
+        Box,
+        Flex,
+      } from "@chakra-ui/react";
+        
+        export default function App() {
+          return (
+            <>
+            <Flex textAlign="center" gap={3} mt={3} direction="column">
+            <Box>
+              <Text>Default</Text>
+              <Slider aria-label="slider-ex-1">
+                <SliderTrack>
+                  <SliderFilledTrack />
+                </SliderTrack>
+                <SliderThumb />
+              </Slider>
+            </Box>
+    
+            <Box>
+              <Text>With xl size</Text>
+              <Slider aria-label="slider-ex-1" size="xl">
+                <SliderTrack>
+                  <SliderFilledTrack />
+                </SliderTrack>
+                <SliderThumb />
+              </Slider>
+            </Box>
+    
+            <Box>
+              <Text>With square variant</Text>
+              <Slider aria-label="slider-ex-1" variant="square">
+                <SliderTrack>
+                  <SliderFilledTrack />
+                </SliderTrack>
+                <SliderThumb />
+              </Slider>
+            </Box>
+          </Flex>
+            </>
+          );
+        }
+      `,
```

Comment:
Can you fix the formatting here? It looks bad on the CodeSandbox:
![image](https://user-images.githubusercontent.com/44305048/210862841-daf28aed-1392-4fa5-bb8f-228dbf8d086b.png)